service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillCreateServiceIndexAction.kt

```diff
@@ -0,0 +1,188 @@
+      )
+    }
+
+    val variant = variantOrBlank.orEmpty().ifBlank { "default" }
+    val label = if (variant == "default") service else "$service ($variant)"
+    val htmlResponseBody = dashboardPageLayout.newBuilder()
+      .title("Create Backfill | Backfila")
+      .breadcrumbLinks(
+        Link("Services", ServiceIndexAction.PATH),
+        Link(
+          label,
+          ServiceShowAction.PATH
+            .replace("{service}", service)
+            .replace("{variantOrBlank}", variantOrBlank ?: ""),
+        ),
+        Link(
+          "Create",
+          PATH
+            .replace("{service}", service)
+            .replace("{variantOrBlank}", variantOrBlank ?: ""),
+        ),
+      )
+      .buildHtmlResponseBody {
+        val registeredBackfills = getRegisteredBackfillsAction.backfills(service, variant)
+
+        PageTitle("Create Backfill") {
+          span("inline-flex shrink-0 items-center rounded-full bg-blue-50 px-2.5 py-0.5 text-s font-medium text-blue-700 ring-1 ring-inset ring-blue-600/20") {
+            val suffix = if (variantOrBlank.isNullOrBlank()) "" else "/$variantOrBlank"
+            +"$service$suffix"
+          }
+        }
+
+        if (registeredBackfills.backfills.isEmpty()) {
+          // TODO add link to runbook?
+          AlertError("No backfills registered for this service. Check docs for how to register backfills.")
+        } else {
+          // If service + variant is set and valid, show registered backfills drop down
+          div("py-10") {
+            ul("grid grid-cols-1 gap-6 sm:grid-cols-2 lg:grid-cols-3") {
+              role = "list"
+
+              registeredBackfills.backfills.map {
+                a {
+                  val variantOrBackfillNameOrId = variantOrBlank.orEmpty().ifBlank { it.name }
+                  href = BackfillCreateAction.PATH
+                    .replace("{service}", service)
+                    .replace("{variantOrBackfillNameOrId}", variantOrBackfillNameOrId)
+                    .replace("{backfillNameOrId}", if (variantOrBackfillNameOrId == it.name) "" else it.name)
+
+                  // TODO make full width
+                  this@ul.li("registration col-span-1 divide-y divide-gray-200 rounded-lg bg-white shadow") {
+                    div("flex w-full items-center justify-between space-x-6 p-6") {
+                      div("flex-1 truncate") {
+                        div("flex items-center space-x-3") {
+                          // Don't include default variant in label, only for unique variants
+//                          val label = if (variant == "default") service else "$service/$variant"
+                          h3("truncate text-sm font-medium text-gray-900") {
+                            +it.name
+                          }
+//                          variant?.let { span("inline-flex shrink-0 items-center rounded-full bg-green-50 px-1.5 py-0.5 text-xs font-medium text-green-700 ring-1 ring-inset ring-green-600/20") { +it } }
```

Comment:
Also here comments that we may want to remove?

Reply:
Deleted